catroid/src/androidTest/java/org/catrobat/catroid/test/formulaeditor/ParserTestOperators.java

```diff
@@ -336,4 +337,24 @@ public void testSubstraction() {
 		second.add(new InternToken(InternTokenType.NUMBER, "1"));
 		FormulaEditorTestUtil.testBinaryOperator(first, Operators.MINUS, second, Double.NaN, testSprite);
 	}
-}
+
+	public void testNegativeZeroEqualsZero() {
+
+		FormulaElement formulaElement = new FormulaElement(ElementType.OPERATOR, Operators.EQUAL.name(), null,
+				new FormulaElement(ElementType.NUMBER, Integer.toString(-0), null), new FormulaElement(ElementType
+				.NUMBER, Integer.toString(0), null));
```

Comment:
Breaking the last `ElementType.NUMBER` at this point is really odd. You can improve the readability by using one FormulaElement per line. Always remember "Code is written once, but read many times".

```java
FormulaElement formulaElement = new FormulaElement(ElementType.OPERATOR, Operators.EQUAL.name(), null,
		new FormulaElement(ElementType.NUMBER, Integer.toString(-0), null),
		new FormulaElement(ElementType.NUMBER, Integer.toString(0), null));
```

Even if it's fine to use `Integer.toString(-0)`, it doesn't add any value compared to "-0".

Please apply to every test.